### PR TITLE
deprecated react/jsx-quotes

### LINKED ISSUE #513: react/jsx-quotes is deprecated. Use jsx-quotes instead
React/jsx-quotes has been deprecated in latest version of eslint.

### PR DESCRIPTION
**Pull request by brad-decker**
replaced this rule with the correct rule. Resolves #513

### CODE REVIEW

**Comment by notxt** [comment]
:+1:

**Comment by chicoxyzzy** [comment]
:+1: this will help to avoid annoying deprecation message

File: packages/eslint-config-airbnb/react.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ module.exports = {
      */
     'react/display-name': 0,         // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/display-name.md
     'react/jsx-boolean-value': 2,    // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-boolean-value.md
-    'react/jsx-quotes': [2, 'double'], // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-quotes.md
+    'jsx-quotes': [2, 'prefer-double'], // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-quotes.md
```
Comment (justjake) [inline packages/eslint-config-airbnb/react.js]: you should change the documentation link to `http://eslint.org/docs/rules/jsx-quotes`

Comment (brad-decker) [inline packages/eslint-config-airbnb/react.js]: Will do, thanks!

**Comment by ljharb** [comment]
This is fixed in #521.

**Pull request closed without merging**